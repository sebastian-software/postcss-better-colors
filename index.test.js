import postcss from "postcss"
import plugin from "."

test("Transform Colors", () => {
  return postcss([ plugin ]).process(".red { color: red; }").then((result) => {
    expect(result.css).toMatchSnapshot()
  })
})
