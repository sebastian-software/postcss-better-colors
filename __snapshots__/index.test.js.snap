// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transform Colors 1`] = `".red { color: red; }"`;
